src/basic/REDUCE_STRUCT-Seq.cpp

```diff
@@ -0,0 +1,139 @@
+ 
+        Real_type xsum = m_init_sum; Real_type ysum = m_init_sum;
+        Real_type xmin = m_init_min; Real_type ymin = m_init_min;
+        Real_type xmax = m_init_max; Real_type ymax = m_init_max;
+
+        for (Index_type i = ibegin; i < iend; ++i ) {
+          REDUCE_STRUCT_BODY;
+        }
+
+        points.SetCenter(xsum/(points.N), ysum/(points.N));
+        points.SetXMin(xmin); 
+        points.SetXMax(xmax);
+        points.SetYMin(ymin);
+        points.SetYMax(ymax);
+        m_points=points;
+
+      }
+      stopTimer();
+
+      break;
+    }
+
+#if defined(RUN_RAJA_SEQ)
+    case Lambda_Seq : {
+
+      auto reduce_struct_x_base_lam = [=](Index_type i) -> Real_type {
+                                   return points.x[i];
+                                 };
+
+      auto reduce_struct_y_base_lam = [=](Index_type i) -> Real_type {
+                                   return points.y[i];
+                                 };
+
+      startTimer();
+      for (RepIndex_type irep = 0; irep < run_reps; ++irep) {
+
+        Real_type xsum = m_init_sum; Real_type ysum = m_init_sum;
+        Real_type xmin = m_init_min; Real_type ymin = m_init_min;
+        Real_type xmax = m_init_max; Real_type ymax = m_init_max; 
+
+        for (Index_type i = ibegin; i < iend; ++i ) {
+          xsum += reduce_struct_x_base_lam(i);
+          xmin = RAJA_MIN(xmin, reduce_struct_x_base_lam(i));
+          xmax = RAJA_MAX(xmax, reduce_struct_x_base_lam(i));
+          ysum += reduce_struct_y_base_lam(i);
+          ymin = RAJA_MIN(ymin, reduce_struct_y_base_lam(i));
+          ymax = RAJA_MAX(ymax, reduce_struct_y_base_lam(i));
+        }
+
+        points.SetCenter(xsum/(points.N), ysum/(points.N));
+        points.SetXMin(xmin); 
+        points.SetXMax(xmax);
+        points.SetYMin(ymin); 
+        points.SetYMax(ymax);
+        m_points=points;
+
+      }
+      stopTimer();
+
+      break;
+    }
+
+    case RAJA_Seq : {
+
+      startTimer();
+      for (RepIndex_type irep = 0; irep < run_reps; ++irep) {
+
+        RAJA::ReduceSum<RAJA::seq_reduce, Real_type> xsum(m_init_sum);
+        RAJA::ReduceSum<RAJA::seq_reduce, Real_type> ysum(m_init_sum);
+        RAJA::ReduceMin<RAJA::seq_reduce, Real_type> xmin(m_init_min);
+        RAJA::ReduceMin<RAJA::seq_reduce, Real_type> ymin(m_init_min);
+        RAJA::ReduceMax<RAJA::seq_reduce, Real_type> xmax(m_init_max);
+        RAJA::ReduceMax<RAJA::seq_reduce, Real_type> ymax(m_init_max);
+
+        RAJA::forall<RAJA::loop_exec>(
+        RAJA::RangeSegment(ibegin, iend), [=](Index_type i) {
+        REDUCE_STRUCT_BODY_RAJA;
+        });
+
+      	points.SetCenter(static_cast<Real_type>(xsum.get()/(points.N)),
+                         static_cast<Real_type>(ysum.get()/(points.N)));
+	points.SetXMin(static_cast<Real_type>(xmin.get())); 
+        points.SetXMax(static_cast<Real_type>(xmax.get()));
+	points.SetYMin(static_cast<Real_type>(ymin.get())); 
```

Comment:
```suggestion
        points.SetXMin(static_cast<Real_type>(xmin.get())); 
        points.SetXMax(static_cast<Real_type>(xmax.get()));
        points.SetYMin(static_cast<Real_type>(ymin.get())); 
```

Reply:
Is this removing tabs in the code?  I attempted to remove them when I saw them. Missed some I guess.